docs/non-normative/groups/system/design-philosophy.md

```diff
@@ -0,0 +1,94 @@
+
+The following sections address some of the most common instrumentation design questions, and how we as a working group have opted to address them. In some cases they are unique to the common semantic conventions guidance due to our unique circumstance, and those cases will be called out specifically.
+
+## Namespaces
+
+Relevant discussions: [\#1161](https://github.com/open-telemetry/semantic-conventions/issues/1161)
+
+The System Semantic Conventions generally cover the following namespaces:
+
+* `system`  
+* `process`  
+* `host`  
+* `memory`  
+* `network`  
+* `disk`  
+* `memory`  
+* `os`
+
+Deciding on the namespace of a metric/attribute is generally informed by the following belief:
+
+**The namespace of a metric/attribute should logically map to the Operating System concept being considered the instrumentation source.**
+
+The most obvious example of this is with language runtime metrics and `process` namespace metrics. Many of these metrics are very similar; most language runtimes provide some manner of `cpu.time`, `memory.usage` and similar metrics. If we were considering de-duplication as the top value in our design, it would follow that `process.cpu.time` and `process.memory.usage` should simply be referenced by any language runtime that might produce those metrics. However, as a working group we believe it is important that `process` namespace and runtime namespace metrics remain separate, because `process` metrics are meant to represent an **OS-level process as the instrumentation source**, whereas runtime metrics represent **the language runtime as the instrumentation source**.
+
+In some cases this is simply a matter of making the instrumentation's purpose as clear as possible, but there are cases where attempts to share definitions across distinct instrumentation sources poses the potential for a clash. The concrete example of a time we accepted this consequence is with `cpu.mode`; the decision was to [unify all separate instances of `*.cpu.state` attributes into one shared `cpu.mode` attributed](https://github.com/open-telemetry/semantic-conventions/issues/1139). The consequence of this is that `cpu.mode` needs to have a broad enum in its root definition, with special exemptions in each different `ref` of `cpu.mode`, since `cpu.mode` used in `process.cpu.time` vs `container.cpu.time` vs `system.cpu.time` etc. has different subsets of the overall enum values. We decided as a group to accept the consequence in this case, however it isn't something we're keen on dealing with all over system semconv, as the instrumentation ends up polluted with so many edge cases in each namespace that it defeats the purpose of sharing the attribute in the first place.
```

Comment:
```suggestion
In some cases this is simply a matter of making the instrumentation's purpose as clear as possible, but there are cases where attempts to share definitions across distinct instrumentation sources poses the potential for a clash. The concrete example of a time we accepted this consequence is with `cpu.mode`; the decision was to [unify all separate instances of `*.cpu.state` attributes into one shared `cpu.mode` attribute](https://github.com/open-telemetry/semantic-conventions/issues/1139). The consequence of this is that `cpu.mode` needs to have a broad enum in its root definition, with special exemptions in each different `ref` of `cpu.mode`, since `cpu.mode` used in `process.cpu.time` vs `container.cpu.time` vs `system.cpu.time` etc. has different subsets of the overall enum values. We decided as a group to accept the consequence in this case, however it isn't something we're keen on dealing with all over system semconv, as the instrumentation ends up polluted with so many edge cases in each namespace that it defeats the purpose of sharing the attribute in the first place.
```

Reply:
ugh, github diff is useless on these long lines 😢

all that I changed above is removing a single character: `cpu.mode` attributed -> `cpu.mode` attribute

Reply:
I hate these things, but I'm more and more thinking we should introduce some markdown lint style enforcement, such as long lines.

Reply:
Made the fix in [487af83](https://github.com/open-telemetry/semantic-conventions/pull/1618/commits/487af83f99a9ab8eeeb937e6cfac28638114cd8b)

Applying line length enforcement in [01f43e9](https://github.com/open-telemetry/semantic-conventions/pull/1618/commits/01f43e943767259ebb72f637b8fe5b428c8a16b0)